src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/Errors/UserStringBuilder.cs

```diff
@@ -656,21 +656,20 @@ private void ErrAppendType(CType pType, SubstContext pctx, bool fArgs)
                             // Add [] with (rank-1) commas inside
                             ErrAppendChar('[');
 
-#if ! CSEE
```

Comment:
I'm guessing the C# expression evaluator. Like here:
[https://github.com/dotnet/corefx/blob/master/src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/Semantics/Types/AggregateType.cs#L22](https://github.com/dotnet/corefx/blob/master/src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/Semantics/Types/AggregateType.cs#L22)

---
In reply to: [109408159](https://github.com/dotnet/corefx/pull/17810#discussion_r109408159) [](ancestors = 109408159)